### PR TITLE
Rename experiment to Accessible Blocks, default to off, add info dialog

### PR DESCRIPTION
**Pull request by shakao**
- Rename experiment to 'Accessible Blocks' for consistency
- Change behavior so experiment enables settings menu item (Accessible Blocks On) which defaults to off
- Add information dialog when toggling accessible blocks on, to explain keyboard commands

Fixes https://github.com/microsoft/pxt-microbit/issues/2779

### CODE REVIEW

File: webapp/src/accessibleblocks.tsx [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+
+    render() {
+        const shown = this.state.shown;
+        if (shown) return <div />;
+        return <sui.Modal isOpen={!this.state.shown} onClose={this.handleOnClose} closeIcon={true}
+            dimmer={true} header={lf("Move, edit, and insert blocks using your keyboard")}>
+            <div className="ui equal width grid">
+                <div className="column">
+                    <h3>{lf("Navigate")}</h3>
+                    <span className="ui small">{lf("Use the WASD keys to move between blocks. Hold shift to move on the workspace.")}</span>
```
Comment (pelikhan) [inline webapp/src/accessibleblocks.tsx:28]: What is the story for non-US keyboards?

Comment (pelikhan) [reply webapp/src/accessibleblocks.tsx:28]: Could we make WASD a localized string and pick keys from there?

Comment (shakao) [reply webapp/src/accessibleblocks.tsx:28]: not handled for now--i think the hope is if they integrate screen reader support we can use the arrow keys for everything

i think the most flexible solution is probably to expose some kind of UI to let users remap the controls based on whatever keyboard they have plugged in, but i'm also hoping we can just switch to arrow keys haha

Comment (abchatra) [reply webapp/src/accessibleblocks.tsx:28]: This is experimentation feature, lets postpone this problem for now. It unrelated to this checkin

**Pull request merged**